src/github.com/getlantern/flashlight/main/settings.go

```diff
@@ -27,15 +29,18 @@ var (
 
 // Settings is a struct of all settings unique to this particular Lantern instance.
 type Settings struct {
-	Version      string
-	BuildDate    string
-	RevisionDate string
-	AutoReport   bool
-	AutoLaunch   bool
-	ProxyAll     bool
-	SystemProxy  bool
-
-	sync.RWMutex
+	UserId    int    `json:"user_id,omitempty"`
+	UserToken string `json:"user_token,omitempty"`
+
+	Version      string `json:"version"`
+	BuildDate    string `json:"build_date"`
+	RevisionDate string `json:"revision_date"`
+	AutoReport   bool   `json:"auto_report"`
+	AutoLaunch   bool   `json:"auto_launch"`
+	ProxyAll     bool   `json:"proxy_all"`
+	SystemProxy  bool   `json:"system_proxy"`
```

Comment:
Yes, it was more to avoid having an uppercase first letter (`AutoReport` instead of `auto_report` or `autoReport`). I'm gonna change them back to camelcase and first lowecase letter.